queries/highlights.scm

```diff
@@ -44,7 +44,18 @@
 (ui_signal_parameter
   (identifier) @variable.parameter)
 
+;;; ui_object_definition
+(ui_object_definition
+  type_name: (identifier) @type)
```

Comment:
This one looks good.

The other rules are redundant since they are included in highlights-javascript.scm.

Reply:
Thanks. .. but how to include scm in scheme?

Reply:
and I have problem in the command of tree-sitter hightlight, how to get the highlight?
![image](https://user-images.githubusercontent.com/60290287/177716647-48c099e5-0475-4ebf-9e53-8a3920d6b04a.png)
I failed..

Reply:
I think that's up to neovim and I don't know.

For example, emacs-tree-sitter has its own highlight rules. So I think there would be a neovim way.
https://github.com/emacs-tree-sitter/tree-sitter-langs/tree/master/queries

Regarding the tree-sitter CLI, .scm files are specified in package.json.

Reply:
> I think that's up to neovim and I don't know.
> 
> For example, emacs-tree-sitter has its own highlight rules. So I think there would be a neovim way. https://github.com/emacs-tree-sitter/tree-sitter-langs/tree/master/queries
> 
> Regarding the tree-sitter CLI, .scm files are specified in package.json.

I mean , I only have the highlights.scm, and if I want to include the scm of javascript , must I copy the text of javascript or I can just use "inculde" to include the scheme?

Reply:
> I mean , I only have the highlights.scm as the entrance, and if I want to include the scm of javascript , must I copy the text of javascript or I can just use "inculde" to include the scheme?

AFAIK, there's no "include" stuff. just copy the content or load multiple .scm files.

> and I have problem in the command of tree-sitter hightlight, how to get the highlight?

https://tree-sitter.github.io/tree-sitter/syntax-highlighting#paths

You need to add tree-sitter-qmljs directory to "parser-directories".

My ~/.config/tree-sitter/config.json:
```
{
  "parser-directories": [
    "/home/yuya/work/2021/tree-sitter",
    "/home/yuya/github",
    "/home/yuya/src",
    "/home/yuya/source"
  ],
  "theme": {
    "attribute": {
      "color": 124,
      "italic": true
    },
    "constant": 94,
    "module": 136,
    "keyword": 56,
    "property": 124,
    "operator": {
      "color": 239,
      "bold": true
    },
    "punctuation.bracket": 239,
    "embedded": null,
    "function.builtin": {
      "bold": true,
      "color": 26
    },
    "variable.parameter": {
      "underline": true
    },
    "number": {
      "color": 94,
      "bold": true
    },
    "variable.builtin": {
      "bold": true
    },
    "function": 26,
    "constant.builtin": {
      "bold": true,
      "color": 94
    },
    "string": 28,
    "constructor": 136,
    "punctuation.delimiter": 239,
    "tag": 18,
    "type": 23,
    "comment": {
      "color": 245,
      "italic": true
    },
    "string.special": 30,
    "type.builtin": {
      "bold": true,
      "color": 23
    }
  }
}

```